packages/graphql-client/src/graphql-client/types.ts

```diff
@@ -26,12 +26,15 @@ export interface GQLExtensions {
   [key: string]: any;
 }
 
-export interface ClientResponse<TData = any> {
+export interface FetchResponseBody<TData = any> {
   data?: Partial<TData>;
-  errors?: ResponseErrors;
   extensions?: GQLExtensions;
 }
 
+export interface ClientResponse<TData = any> extends FetchResponseBody<TData> {
+  errors?: ResponseErrors;
+}
+
```

Comment:
@melissaluu nit-y question: do you think these types should be in the API utilities instead of in graphql-client?

Reply:
Ok, thanks!